internal/recommender/metrics/provider.go

```diff
@@ -0,0 +1,100 @@
+package metrics
+
+import (
+	"context"
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/api"
+	"github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+	"github.com/scylladb/go-log"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"net/url"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+const (
+	appLabel       = "prometheus"
+	componentLabel = "server"
+	releaseLabel   = "scylla-prom"
```

Comment:
I think these should be configurable, i bet noone is using `scylla-prom` as a release label.